app/assets/javascripts/polaris_view_components.js

```diff
@@ -176,41 +176,47 @@ function formatBytes(bytes, decimals) {
 class Autocomplete extends Controller {
   static targets=[ "popover", "input", "results", "option", "emptyState" ];
   static values={
+    multiple: Boolean,
     url: String,
     selected: Array
   };
-  connect() {
-    this.inputTarget.addEventListener("input", this.onInputChange);
-  }
-  disconnect() {
-    this.inputTarget.removeEventListener("input", this.onInputChange);
+  onFocus() {
+    this.showAutocomplete();
   }
-  toggle() {
```

Comment:
I renamed this to `showAutocomplete` as this wasn't really "toggling", but only displaying/showing the autocomplete suggestions